bin/ubuntu-core-initramfs

```diff
@@ -739,35 +740,28 @@ def create_efi(parser, args):
             except gzip.BadGzipFile:
                 pass
 
-    # kernel.efi sections will be aligned to the page size by llvm-objcopy
-    # (which is the default value for SectionAlignment in the PE header).
-    # --set-section-flags are such so added COFF sections have flags:
-    # CONTENTS, ALLOC, LOAD, READONLY, DATA
-    # (see llvm-objcopy man page for details)
-    # To see more details, run 'objdump --all-headers kernel.efi'
-    # (objdump gives a bit more of information than llvm-objdump in this case)
-    objcopy_cmd = [
-            "llvm-objcopy",
+    # TODO: add --splash
+    ukify_cmd = [
+        '/usr/lib/systemd/ukify', 'build',
+        '--linux', args.kernel,
+        '--initrd', args.initrd,
+        '--output', args.output,
+        '--os-release', '@{}'.format(args.os_release),
```

Comment:
Is this simply adding a section with the content of the file or is there something else to this? What is it used for?